### PR TITLE
Remove the "old" loging, as mentioned in #97

### PR DESCRIPTION
**Pull request by zzeroo**
@antoyo to be honest. I have not looked at the new relm error story yet.
Is there anything else I can do here?

### CODE REVIEW

**Comment by antoyo** [comment]
Could you please remove anything related to logging, like the crates `env_logger` and `log`?
Thanks for your contribution!

File: relm-attributes/Cargo.toml [hunk 1]
```diff
@@ -8,10 +8,15 @@ repository = "https://github.com/antoyo/relm"
 version = "0.14.3"
 
 [dependencies]
+<<<<<<< HEAD
```
Comment (Eijebong) [inline relm-attributes/Cargo.toml]: Rebase fail :p

**Comment by zzeroo** [comment]
@all sorry for the noise I'm tired >< 

@antoyo There is only one log crate usage in `relm-state`. But this looks good for me.

**Comment by antoyo** [comment]
I'll take a look at this error.
It looks similar to the one we had after merging PR #104.

**Comment by Eijebong** [comment]
@antoyo I have a patch, I'll open a PR

**Comment by antoyo** [comment]
@zzeroo Please rebase against master (I've just merged @Eijebong PR).

**Comment by antoyo** [comment]
Thank you for your contribution.

**Pull request merged**